### ISSUE TITLE
[2022 US Symposium] Smoking Conventions

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Themis#65]
TYPE | NOTES
-- | --
ITEM | How should we handle the capture of smoking in the CDM?
FORUM POST | https://forums.ohdsi.org/t/2022-us-symposium-smoking-conventions/17500
SOLUTION | ITBD
NEXT STEPS | TBD

**Comment by cgreich** [comment]
We have a proposal. I think it is ready. Should we bring it up next week at the Symposium Workshop?

**Comment by clairblacketer** [comment]
@cgreich yes please. Melanie will be leading this one, can you send us your proposal so we can bring it to the group?

**Comment by MelaniePhilofsky** [comment]
This issue needs documentation for ETL guidance and end user guidance.
